mappings/net/minecraft/client/render/entity/EntityRenderDispatcher.mapping

```diff
@@ -1,6 +1,45 @@
 CLASS net/minecraft/class_578 net/minecraft/client/render/entity/EntityRenderDispatcher
+	FIELD field_2489 INSTANCE Lnet/minecraft/class_578;
+	FIELD field_2490 offsetX D
+	FIELD field_2491 offsetY D
+	FIELD field_2492 offsetZ D
+	FIELD field_2493 textureManager Lnet/minecraft/class_76;
+	FIELD field_2494 heldItemRenderer Lnet/minecraft/class_556;
+	FIELD field_2495 world Lnet/minecraft/class_18;
+	FIELD field_2496 livingEntity Lnet/minecraft/class_127;
```

Comment:
this is the camera entity. So matching it to [camera](https://maven.fabricmc.net/docs/yarn-1.21.1+build.3/net/minecraft/client/particle/ParticleManager.html#addBlockBreakParticles(net.minecraft.util.math.BlockPos,net.minecraft.block.BlockState)) would make it `cameraEntity`